launcher-docker-compose.yaml

```
@@ -0,0 +1,17 @@
+version: '3.8'
+
+services:
+  web:
+    image: barakeinavhnear/launcher:latest # todo (security) use spesific image hash  
```

I created an issue for this: https://github.com/near/mpc/issues/531

```suggestion
    image: barakeinavhnear/launcher:latest # todo(#531) use specific image hash  
```